### PR TITLE
Implement the update local orders callback method

### PR DESCRIPTION
**Pull request by deivamagalhaes**
# Summary

This PR implements the `Commerce\Orders::update_local_orders()` method.

## QA

- [x] Integrations test pass

## Before merge

- [ ] I have confirmed these changes in each supported minor WooCommerce version

### CODE REVIEW

File: includes/Commerce/Orders.php [hunk 1]
```diff
@@ -96,7 +97,55 @@ public function update_local_order( Order $remote_order, \WC_Order $local_order
 	 */
 	public function update_local_orders() {
 
-		// TODO: implement
+		$page_id = facebook_for_woocommerce()->get_integration()->get_facebook_page_id();
+
+		try {
+
+			$response = facebook_for_woocommerce()->get_api()->get_new_orders( $page_id );
```
Comment (ChaseWiseman) [inline includes/Commerce/Orders.php:104]: We'll need to pass in the page access token for this API request, which we can now after implementing [this story](https://app.clubhouse.io/skyverge/story/62622/update-the-base-api-getter-to-accept-an-access-token)

File: includes/Commerce/Orders.php [hunk 2]
```diff
@@ -96,7 +97,55 @@ public function update_local_order( Order $remote_order, \WC_Order $local_order
...
+			} catch ( \Exception $exception ) {
+
+				if ( ! empty( $local_order ) ) {
+					// add note to order
+					$local_order->add_order_note( 'Error updating local order from Commerce order from the Orders API: ' . $exception->getMessage() );
+				} else {
+					facebook_for_woocommerce()->log( 'Error creating local order from Commerce order from the Orders API: ' . $exception->getMessage() );
+				}
+
+				return;
```
Comment (ChaseWiseman) [inline includes/Commerce/Orders.php:136]: I think this should be `continue` to keep processing remaining orders, right?

File: includes/Commerce/Orders.php [hunk 3]
```diff
@@ -96,7 +97,55 @@ public function update_local_order( Order $remote_order, \WC_Order $local_order
...
+			}
+
+			if ( ! empty( $local_order ) && Order::STATUS_CREATED === $remote_order->get_status() ) {
+
+				// acknowledge the order
+				try {
+					facebook_for_woocommerce()->get_api()->acknowledge_order( $remote_order->get_id(), $local_order->get_id() );
```
Comment (ChaseWiseman) [inline includes/Commerce/Orders.php:143]: this will also need the page access token

File: includes/Commerce/Orders.php (hunk 1 above)
Comment (deivamagalhaes) [reply includes/Commerce/Orders.php]: Fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1520/commits/5c72c5e8b4d3b62a9761eddd85f85824051f82e9

File: includes/Commerce/Orders.php (hunk 3 above)
Comment (deivamagalhaes) [reply includes/Commerce/Orders.php]: Fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1520/commits/5c72c5e8b4d3b62a9761eddd85f85824051f82e9

**Pull request merged**